scriptmodules/supplementary/raspbiantools.sh

```diff
@@ -25,6 +25,14 @@ function apt_upgrade_raspbiantools() {
 function lxde_raspbiantools() {
     aptInstall --no-install-recommends xorg lxde
     aptInstall raspberrypi-ui-mods rpi-chromium-mods gvfs
+    # On `buster`, disable PulseAudio since it messes up the audio settings
+    # remove the lxpanel plugin for PulseAudio volume, to prevent a crash due to missing PulseAudio
+    #  and install the volume lxpanel plugin that supports ALSA
+    if compareVersion "$__os_debian_ver" lt 11; then
```

Comment:
Please can you use `[[ "$__os_debian_ver -lt 11]]` - see 61c46aa0055051aab9dad5695994cf3d8bad3e42